packages/lending/src/protocols/aave-v2/lending-protocol.ts

```diff
@@ -79,6 +84,17 @@ export class LendingProtocol extends Protocol {
     this.reserveMap = reserveMap;
   }
 
+  async getReserveTokens() {
```

Comment:
Do we need a return type here?

Reply:
We can skip the return type since TypeScript can infer it